src/consumer/frame.js

```diff
@@ -141,6 +149,26 @@ class Frame extends EventEmitter {
     if (this.wrapper.parentNode === this.container) {
       this.container.removeChild(this.wrapper);
       this.emit('xfc.unmounted');
+      this.cleanup();
+    }
+  }
+
+  /**
+   * Cleans up references of detached nodes by setting them to null
+   * to avoid potential memory leak
+   */
+  cleanup() {
+    this.iframe = null;
+    this.wrapper = null;
+  }
+
+  /**
+   * Loads a new page within existing frame.
+   * @param  {string} url - the URL of new page to load.
+   */
+  load(url) {
+    if (url) {
+      this.iframe.src = url;
```

Comment:
Don't you have to set `this.origin` to the origin of the new url if they differ? I suggest move the `[loadPage](https://github.com/cerner/xfc/blob/reuse-iframe/src/consumer/frame.js#L83-L89)` logic into the `load()` method and having `loadPage` call this instead.

Reply:
I'd make sure we test this with loading a new src from a different origin

Reply:
Good call! I've moved the logic of `loadPage` to `load` method and had `loadPage` call `load` instead.
As for test, we do have a [test page](https://github.com/cerner/xfc/blob/master/example/cross_origin_communication/3_c_index.html#L63) in example that shows how provider triggers an loadPage event to load a new url with a different origin (from `localprovider.com` to `localhost`). Since we've made `loadPage` call `load` method, I think that may be good to cover tests for `load` as well.